packages/cli/src/ceramic-daemon.ts

```diff
@@ -349,25 +351,30 @@ export class CeramicDaemon {
     const adminPinsRouter = ErrorHandlingRouter(this.diagnosticsLogger)
     const legacyPinsRouter = ErrorHandlingRouter(this.diagnosticsLogger)
     const adminNodeStatusRouter = ErrorHandlingRouter(this.diagnosticsLogger)
-
-    app.use('/api/v0', baseRouter)
-    baseRouter.use('/commits', commitsRouter)
-    baseRouter.use('/documents', documentsRouter)
-    baseRouter.use('/multiqueries', multiqueriesRouter)
-    baseRouter.use('/node', nodeRouter)
-    baseRouter.use('/records', recordsRouter)
-    baseRouter.use('/streams', streamsRouter)
-    baseRouter.use('/collection', collectionRouter)
-    baseRouter.use('/admin/getCode', adminCodesRouter)
-    baseRouter.use('/admin/models', adminModelRouter)
-    baseRouter.use('/admin/status', adminNodeStatusRouter)
+    const v1AdminModelRouter = ErrorHandlingRouter(this.diagnosticsLogger)
+
+    app.use('/api', apiRouter)
+    apiRouter.use('/v0', v0Router)
+    v0Router.use('/commits', commitsRouter)
+    v0Router.use('/documents', documentsRouter)
+    v0Router.use('/multiqueries', multiqueriesRouter)
+    v0Router.use('/node', nodeRouter)
+    v0Router.use('/records', recordsRouter)
+    v0Router.use('/streams', streamsRouter)
+    v0Router.use('/collection', collectionRouter)
+    v0Router.use('/admin/getCode', adminCodesRouter)
+    v0Router.use('/admin/models', adminModelRouter)
+    v0Router.use('/admin/status', adminNodeStatusRouter)
     // Admin Pins Validate JWS Middleware
-    baseRouter.use('/admin/pins', this.validateAdminRequest.bind(this))
-    baseRouter.use('/admin/pins', adminPinsRouter)
+    v0Router.use('/admin/pins', this.validateAdminRequest.bind(this))
+    v0Router.use('/admin/pins', adminPinsRouter)
+
+    apiRouter.use('/v1', v1Router)
```

Comment:
I'd like to pull in @oed and @ukstv to confirm the use of v1 http APIs in this way.  We haven't yet ever tried to increase our HTTP api version, and I'm not sure if best practice would be to update all of our APIs at once vs having some apis on v0 while others are on v1

Reply:
I'm not completely sure what best practice is either, but it feels a bit strange to use both `v0` and `v1` at the same time.

Reply:
There's a few ways to do it, one being slowly move over commands that are needed, updating clients as needed to the new methods, calling v0 until v1 api's are available. Another is to bring over all v0 api's except the ones we want to replace.

Reply:
See my comment below. I don't think we need to introduce a breaking change here at all?
Shouldn't we just be able to make the `v0` of the api parse the new and the old formatted JWS?